source/support.html.erb

```diff
@@ -134,7 +138,24 @@ title: Support
 
       <p>
         Outside of working hours, please use the escalation contact
-        details you have been sent.
+        details you have been sent.</p>
+
+        <h2 class="heading-large">Keeping up to date</h2>
+
+        <p>
+        Our <a href="https://status.cloud.service.gov.uk/">system status page</a>
+        shows if live applications and databases are available. You can use this
+        page to sign up to get alerts and incident updates.
+      </p>
+
+      <p>
+        <a href="https://groups.google.com/a/digital.cabinet-office.gov.uk/forum/?hl=en#!forum/gov-uk-paas-announce">Join our announcements mailing list</a>
+        to find out about new features and changes to the platform. If you have
+        trouble signing up, please contact
+        gov-uk-paas-support@digital.cabinet-office.gov.uk and we will add you
```

Comment:
Recommend adding in email address link - apologies for missing this first time around.